packages/config-loader/src/loader.ts

```diff
@@ -126,8 +126,16 @@ export async function loadConfig(
     .filter((e): e is { url: string } => e.hasOwnProperty('url'))
     .map(configTarget => configTarget.url);
 
-  if (remote === undefined && configUrls.length > 0) {
-    throw new Error(`Remote config detected but this feature is turned off`);
+  if (remote === undefined) {
+    if (configUrls.length > 0) {
+      throw new Error(
+        `Please make sure you are passing the remote option when loading the configuration. See https://backstage.io/docs/conf/writing#configuration-files for detailed info.`,
```

Comment:
"...when loading remote configurations..." or something like that perhaps? just wanna avoid confusion

Reply:
Fixed.